src/Root.cpp

```diff
@@ -159,6 +159,14 @@ void cRoot::Start(std::unique_ptr<cSettingsRepositoryInterface> a_OverridesRepo)
 	m_MojangAPI = new cMojangAPI;
 	bool ShouldAuthenticate = settingsRepo->GetValueSetB("Authentication", "Authenticate", true);
 	m_MojangAPI->Start(*settingsRepo, ShouldAuthenticate);  // Mojang API needs to be started before plugins, so that plugins may use it for DB upgrades on server init
+	AString MaxLoadedChunks = settingsRepo->GetValueSet("Server", "MaxChunkRAM", "30");
+	m_MinimizeRam = settingsRepo->GetValueSetB("Server", "MinimizeRAM", true);
```

Comment:
The MinimizeRAM flag modifies the behavior of the chunk saving scheme and isn't just about setting `MaxChunkRam` to 0. But I am not sure I want to keep it, it does sound confusing.